<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Multi_Config_Window</class>
 <widget class="QMainWindow" name="Multi_Config_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new_config"/>
   <addaction name="action_apply_config"/>
   <addaction name="separator"/>
   <addaction name="action_add_motor"/>
  </widget>
  <action name="action_new_config">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New Config</string>
   </property>
  </action>
  <action name="action_apply_config">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/upload.png</normaloff>:/icons/upload.png</iconset>
   </property>
   <property name="text">
    <string>Apply Config</string>
   </property>
  </action>
  <action name="action_add_motor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/add_motor.png</normaloff>:/icons/add_motor.png</iconset>
   </property>
   <property name="text">
    <string>Add Motor</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
